src/components/editor/editor-pane/editor-pane.tsx

```diff
@@ -61,12 +64,53 @@ const onChange = (editor: Editor) => {
   }
 }
 
+interface PasteEvent {
+  clipboardData: {
+    files: FileList
+  },
+  preventDefault: () => void
+}
+
+const onPaste = (pasteEditor: Editor, event: PasteEvent) => {
+  if (event && event.clipboardData && event.clipboardData.files) {
+    event.preventDefault()
+    const files: FileList = event.clipboardData.files
+    if (files && files.length >= 1) {
+      handleUpload(files, pasteEditor)
+    }
+  }
+}
+
+interface DropEvent {
+  pageX: number,
+  pageY: number,
+  dataTransfer: {
+    files: FileList
+  }
+  preventDefault: () => void
+}
+
+const onDrop = (dropEditor: Editor, event: DropEvent) => {
+  if (event && event.pageX && event.pageY && event.dataTransfer && event.dataTransfer.files) {
+    event.preventDefault()
+    const top: number = event.pageY
+    const left: number = event.pageX
+    const newCursor = dropEditor.coordsChar({ top, left }, 'page')
+    dropEditor.setCursor(newCursor)
```

Comment:
As the cursor is set just after the drop occured, this would not bring anything at this point. We could however try to change the cursor position constantly while dragging and see if this helps to orient the user.

Reply:
I implemented this, but the cursor is still mostly invisible and I'm not sure how to fix that yet.

Reply:
Maybe this is useful in any kind?
https://discuss.codemirror.net/t/how-to-always-display-cursor/227

Reply:
done